packages/codemods/meteor/3/mongo-db-async-methods/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "mongo-db-async-methods-transformation",
+  "license": "MIT",
+  "devDependencies": {
```

Comment:
Please include `"private": true` if you don't want to publish the codemod to npm